tools/mo/openvino/tools/mo/ops/convolution.py

```diff
@@ -287,3 +290,26 @@ def reverse_infer(node: Node):
                     break
             if shape is not None:
                 node.in_port(0).data.set_shape(shape)
+
+    @staticmethod
+    def type_infer(node):
```

Comment:
This code doesn't work in any known to me model, but I suspect on broader model scope it can be working and it is relatively safe to change type of conv constant.

Reply:
I disagree to have dead code here with basically no tests and no known use cases. We had problems with `type_infer` due to inconsistent types in constants which you addressed via `mo_array` everywhere. Lets get rid of it here and in the matmul too

Reply:
this is not a dead code, but a safety bag. It may work in unexpected cases